man/rgbasm.1

```diff
@@ -167,7 +167,7 @@ This acts as if a
 was read before the input
 .Ar asmfile .
 .It Fl p Ar pad_value , Fl Fl pad-value Ar pad_value
-Use this as the fill value for
+Use this as the value for
```

Comment:
See this thread: https://github.com/gbdev/rgbds/commit/80bdc889f86796307bf61556107505ae819e7775#r131394569

Reply:
Copying it to here for future:

> > > > "Use this as the default value"?
> > > 
> > > We already say "The default is 0x00" below. I think having a default value for a default value is a bit confusing.
> > > 
> > > How about just "Use this as the value"?
> > 
> > "...directives in ROM sections, unless overridden.", maybe?
> 
> I copied the phrasing for `-Q`: "Use this as the precision of fixed-point numbers after the decimal point, unless they specify their own precision."

But yeah, I think the current phrasing is OK.

Reply:
Idk, it's the "own value(s)" phrasing that I feel doesn't jive well with `ds`. Perhaps passive voice would work better? "...unless values are explicitly specified"?

Reply:
Scratch that: I couldn't see the "overridden" change outside of the Files view; this sounds perfect!